test.py

```diff
@@ -5,24 +5,34 @@
 
 callCount = 0
 
+
 @app.get("/")
 async def h():
     global callCount
-    callCount +=  1
+    callCount += 1
     message = "Called " + str(callCount) + " times"
     return message
 
+
+@app.post("/post")
+async def postreq(body):
+    return bytearray(body).decode("utf-8")
+
+
 @app.get("/sleep")
 async def sleeper():
     await asyncio.sleep(5)
     return "sleep function"
 
+
 @app.get("/blocker")
 def blocker():
     import time
+
     time.sleep(10)
     return "blocker function"
 
-if __name__=="__main__":
-    app.start(port=5000)
 
+if __name__ == "__main__":
+    app.add_header("server", "robyn")
```

Comment:
shouldn't we be adding headers in a request instead of adding it in an app?

Reply:
Possibly be helpful to have both for sure, people tend to set global ones at the start, and add others per request